### PR TITLE
Tailwind init

### PR DESCRIPTION
**Pull request by meridithv**
Tailwind added and integrated, some basic typeface/color and layout set-ups

### CODE REVIEW

File: client/features/navbar/Navbar.js [hunk 1]
```diff
@@ -14,24 +15,38 @@ const Navbar = () => {
...
+                className="hover:text-tahiti"
+              >
+                LOGOUT
+              </button>
+            </div>
+          ) : (
+            <div className="hidden md:flex space-x-6 text-sm uppercase font-plex tracking-widest">
+              {/* The navbar will show these links before you log in */}
+              <Link to="/login">Login</Link>
+              <Link to="/signup">Sign Up</Link>
```
Comment (blakebequette) [inline client/features/navbar/Navbar.js:46]: will want our NavBar to have /home & /products even if they're a guest; maybe same as LoggedIn except the logout changes to login & register buttons

File: public/index.html [hunk 2]
```diff
@@ -4,11 +4,28 @@
...
+    <body class="snow-bg bg-fixed overscroll-none">
+      <div class="bg-gray-50">
+        <div class="max-w-7xl mx-auto grid grid-cols-12 bg-gray-200">
+             <div class="col-span-2">
+               <!--Left sidebar -->
+             </div>
+             <div class="col-span-8 bg-ecru min-h-screen">
+<!-- Navbar -->
+          <div class="mx-auto px-0 pt-8">
+            <div id="app"></div>
```
Comment (blakebequette) [inline public/index.html:21]: normally would make the containers / sidebar react components - but i'm betting this is the tailwind design pattern & I trust you so it looks good to me

File: public/public_images_placeholder-logo_png.bundle.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+(self["webpackChunkpostico_gracehopper"] = self["webpackChunkpostico_gracehopper"] || []).push([["public_images_placeholder-logo_png"],{
+
+/***/ "./public/images/placeholder-logo.png":
+/*!********************************************!*\
+  !*** ./public/images/placeholder-logo.png ***!
+  \********************************************/
+/***/ (() => {
+
```
Comment (blakebequette) [inline public/public_images_placeholder-logo_png.bundle.js:8]: will want this in the .gitignore too

File: package.json [hunk 4]
```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "build": "webpack",
+    "build:tailwind": "tailwindcss -i public/input.css -o public/output.css",
```
Comment (blakebequette) [inline package.json:8]: if you're doing build:tailwind every time something changes in the css, we may want a script command that combines build:dev with build:tailwind

**Review by blakebequette — APPROVED** [review]
Looks good - like the pictures too.

File: client/features/navbar/Navbar.js (hunk 1 above)
Comment (meridithv) [reply client/features/navbar/Navbar.js:46]: Thank you! Fixed in this next push. Do we want /home and /products to be viewable by guests? Right now that's not possible (see AppRoutes.js) but I believe it's a simple change

Comment (meridithv) [reply client/features/navbar/Navbar.js:46]: At present I made the HOME and PRODUCTS buttons on the NavBar visible, but click on them and you'll continue to be prompted for a login

File: public/index.html (hunk 2 above)
Comment (meridithv) [reply public/index.html:21]: Thank you! We can always revisit later if this proves annoying

File: public/public_images_placeholder-logo_png.bundle.js (hunk 3 above)
Comment (meridithv) [reply public/public_images_placeholder-logo_png.bundle.js:8]: Yes I'm abandoning trying to put a locally hosted image in a JSX component for now. We can do it, but it requires changing the webpack config and I don't think it's worth it. There are likely easier ways

File: package.json (hunk 4 above)
Comment (meridithv) [reply package.json:8]: something like this? `"build:devtail": "npm run build:dev && npm run build:tailwind"`

File: public/input.css [hunk 5]
```diff
@@ -0,0 +1,203 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
```
Comment (Irwinb14) [inline public/input.css:4]: Everything looks good! This file is where you can make style classes that can be reused on JSX elements. 
A style class for a button would look like --> 
.btn{
    @apply w-full block text-sm rounded-full border-double border-2 border-purple-500 hover:bg-slate-300;
}

and then to apply it in the component you would use --> class="btn"

File: package.json (hunk 4 above)
Comment (blakebequette) [reply package.json:8]: yes exactly

File: client/features/navbar/Navbar.js (hunk 1 above)
Comment (blakebequette) [reply client/features/navbar/Navbar.js:50]: yes i think we should change to visible

File: package.json (hunk 4 above)
Comment (meridithv) [reply package.json:8]: done and thanks!

**Pull request merged**